### DISCUSSION TITLE
What should I set with AIRFLOW__SENTRY__BEFORE_SEND param in airflow docker-compose?

### DISCUSSION THREAD

**Discussion by yuanke7** [apache/airflow#20192] [Q&A]
I'm trying to upgrade Airflow version from 2.1.2 to 2.2.0.

I use command docker-compose up airflow-init, and it failed.

Error shows that I have to set AIRFLOW__SENTRY__BEFORE_SEND param, btw I have set param AIRFLOW__SENTRY__SENTRY_ON and AIRFLOW__SENTRY__SENTRY_DSN in version 2.1.2 and it works fine with my sentry.

What param should I set on this section? Pls help, thanks~

**Comment by yuanke7** [comment]
I read doc but it says default to None which is not work.

**Reply by potiuk** [reply]
See https://github.com/apache/airflow/issues/18963 - there is a workaround - and it's going to be fixed permanently in the next airflow release.

**Reply by potiuk** [reply]
Actually, correction. It's already been fixed in 2.2.1

**Reply by potiuk** [reply]
I heartily recommend to migrate to 2.2.2 . There are more things fixed in 2.2.1 and 2.2.2

**Reply by yuanke7** [reply]
> I heartily recommend to migrate to 2.2.2 . There are more things fixed in 2.2.1 and 2.2.2

Thanks lot !!!

**Reply by yuanke7** [reply]
> I heartily recommend to migrate to 2.2.2 . There are more things fixed in 2.2.1 and 2.2.2

Sorry but my issue not fix yet ...  https://github.com/apache/airflow/issues/18422
